pkgs/development/tools/ilspy/default.nix

```diff
@@ -0,0 +1,56 @@
+{ pkgs ? (import <nixpkgs> {})}:
+with pkgs;
+let 
+  libs = [
+    xorg.libX11
+    stdenv.cc.cc.lib
+    kerberos
+    curl
+    lttng-ust
+    gtk3
+    openssl.out
+    icu
+  ];
+in
+stdenv.mkDerivation rec {
+  pname = "ilspy";
+  version = "unstable-5.0-rc2";
+
+  nativeBuildInputs = [ 
+    autoPatchelfHook
+  ];
+
+  dontStrip = true;
+  sourceRoot = ".";
+
+  buildInputs = [
+    makeWrapper
+    unzip
+  ] ++ libs;
+
+  src = fetchurl {
+    url = "https://github.com/icsharpcode/AvaloniaILSpy/releases/download/v${version}/ILSpy-linux-x64-Release.zip";
+    sha256 = "1p1fkdkpbcqj2q4n7y241lzx98w33vmh4n2gahw7lbrrg1p7rxxa";
+  };
+
+  installPhase = ''
+    chmod a+x ILSpy
+    mkdir -p $out/bin $out/share/applications
+    cp ${./ILSpy.ico} $out/share/ILSpy.ico
+    mv * $out/bin
+
+    wrapProgram $out/bin/ILSpy \
+      --prefix LD_LIBRARY_PATH : "${stdenv.lib.makeLibraryPath libs}"
+
+    substituteAll ${./ILSpy.desktop} $out/share/applications/ILSpy.desktop    
+
+  '';
+
+  meta = with lib; {
+    description = ".NET assembly browser and decompiler.";
```

Comment:
The `description` shouldn't end in a period (see [here](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes)).

Reply:
done in 2d064c120915267b65d41a1d9f3ddd51021db4da